packages/rspack-cli/src/rspack-cli.ts

```diff
@@ -147,6 +152,21 @@ export class RspackCLI {
 					}
 				});
 			}
+			if (process.env.RSPACK_PROFILE) {
+				resolveProfile(process.env.RSPACK_PROFILE).forEach(profileOptions => {
+					if (profileOptions.kind === "rust trace") {
```

Comment:
It seems that spaces are rarely seen, usually connected with underscores or dashes?